HelpAssistHer/help-assist-her
# Connect Out of Business button to backend

## Description
Connect the Out of Business button to the backend so that the resource is updated when the Out of Business button is clicked.

## Test Plan
1. Click Verify Next Resource
1. Click Out of Business
1. Look in the network tab and verify a request was sent to the `out-of-business` endpoint with a payload of `outOfBusiness: true`. The response should be the updated resource.
1. Find the resource in the database and verify it has `outOfBusiness: true`
1. Verify that when the button was clicked, the button turns orange, an orange overlay is over the form, and form fields are not editable
1. Click the Out of Business button again and verify another request is sent with `outOfBusiness: false` and that the button/overlay is toggled
1. Modify a few resources in the database to be `outOfBusiness: true`. 
1. Click Verify Next Resource until a resource with `outOfBusiness: true` populates the form.
1. Verify that the button turns orange and the orange overlay appears when that resource is loaded.
